src/Cli/Commands/AddTelemetryOptions.cs

```diff
@@ -68,6 +80,30 @@ public AddTelemetryOptions(
         [Option("otel-service-name", Default = "dab", Required = false, HelpText = "(Default: dab) Headers for Open Telemetry for telemetry data")]
         public string? OpenTelemetryServiceName { get; }
 
+        // To specify whether Azure Log Analytics telemetry should be enabled. This flag is optional and default value is false.
+        [Option("azure-log-analytics-enabled", Default = CliBool.False, Required = false, HelpText = "(Default: false) Enable/Disable Azure Log Analytics")]
```

Comment:
The correct syntax should be `dab configure --runtime.telemetry.azure-log-analytics.enabled true` please.

Reply:
@RubenCerna2079, the issue explicitly mentioned we should use `dab configure`. Even if current telemetry options are being added using `add-telemetry`, we should implement what the issue expects. Please create a separate PR for dab configure command edits as described in the issue #2709